packages/cli/commands/accounts/list.js

```diff
@@ -53,21 +56,28 @@ const sortAndMapPortals = portals => {
 
 const getPortalData = mappedPortalData => {
   const portalData = [];
-  Object.values(mappedPortalData).forEach(set => {
+  Object.entries(mappedPortalData).forEach(([key, set]) => {
+    const hasParentPortal = set.filter(
+      p => p.portalId === parseInt(key, 10)
+    )[0];
     set.forEach(portal => {
       let name = portal.name;
       if (isSandbox(portal)) {
         name = `${portal.name} ${getSandboxName(portal)}`;
-        if (set.length > 1) {
+        if (hasParentPortal && set.length > 1) {
           name = `↳ ${name}`;
         }
       } else if (isDeveloperTestAccount(portal)) {
         name = `${portal.name} [${
           HUBSPOT_ACCOUNT_TYPE_STRINGS[HUBSPOT_ACCOUNT_TYPES.DEVELOPER_TEST]
         }]`;
-        if (set.length > 1) {
+        if (hasParentPortal && set.length > 1) {
```

Comment:
If multiple portals referenced the same parent portal ID but that parent portal didn't exist in the config, it would display incorrect name indentations

Before:
<img width="771" alt="Screenshot 2024-04-03 at 14 36 07" src="https://github.com/HubSpot/hubspot-cli/assets/16788677/979e95b8-65ae-47a8-adb2-2c6efc6e736d">

After:
<img width="755" alt="Screenshot 2024-04-03 at 14 35 59" src="https://github.com/HubSpot/hubspot-cli/assets/16788677/aca178ec-096b-478e-86a9-390f6a731c6b">